packages/core/src/lib/template/template.interface.ts

```diff
@@ -11,7 +11,7 @@ export interface IMessageValidator {
 }
 
 export interface IMessage {
-  subject?: string;
+  subject?: string | ((config: IMessage) => string);
```

Comment:
```suggestion
  subject?: string | ((payload: ITriggerPayload) => string);
```
We are passing the payload as the first parameter, in case customizations needed to be done. What was your usecase for passing the IMessage itself?

Reply:
From my side no specific use case at all. I miss understood what we spoke about in discord then